ICAFIX/hcp_fix_multi_run

```diff
@@ -352,6 +360,10 @@ then
 	log_Err_Abort "--fallback-threshold is not allowed when --vol-wisharts=1 is used"
 fi
 
+if [[ ${FSL_FIX_MATLAB_MODE} != 1 ]] && [[ ${ICAmethod} == ICASSO ]];then
```

Comment:
We should probably remove this check and recompile.

Reply:
Also, this would have prevented interpreted octave from trying it, so not a good test condition, even if we wanted to not recompile for some reason.